src/PythonQtImporter.cpp

```diff
@@ -289,14 +289,17 @@ PythonQtImporter_load_module(PyObject *obj, PyObject *args)
       // The package attribute is needed to resolve the package name if it is referenced as '.'. For example,
       // when importing the encodings package, there is an import statement 'from . import aliases'. This import
       // would fail when reloading the encodings package with importlib.
-      err = PyDict_SetItemString(dict, "__package__", PyUnicode_FromString(fullname));
+      PyObject* fullnameObj = PyUnicode_FromString(fullname);
+      err = PyDict_SetItemString(dict, "__package__", fullnameObj);
+      Py_XDECREF(fullnameObj);
       if (err != 0) {
         Py_DECREF(code);
         Py_DECREF(mod);
         return NULL;
       }
 #endif
     }
+    Py_DECREF(mod);
```

Comment:
Is this correct? The documentation states that PyModule_AddModule returns a borrowed reference. But I see that Py_DECREF is also used on mod before, so it seems as if this is the correct way to do this...

Reply:
Yes, good catch, I probably wrongly assumed the existing code was correct but you are right, documentation says it returns a borrowed reference so I removed all the DECREFs.